assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -54,9 +57,16 @@ const Header = styled.h2`
   cursor: pointer;
 `;
 
-const Heading = styled(Headline)`
-  color: ${({ theme }) => theme.colors.fg.secondary};
-  line-height: 32px;
+const Heading = styled.span`
+  color: ${({ theme }) => theme.colors.fg.primary};
+  ${({ theme }) =>
+    themeHelpers.expandPresetStyles({
```

Comment:
The `THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.XX_SMALL` is now assigned twice to this text -- once from the `Header` and once here -- should it be in one place only?

Reply:
yeah let me see if I can fix that